crates/primitives/transactions/src/execution.rs

```diff
@@ -547,8 +551,7 @@ impl Execute for L1HandlerTransaction {
         block_context: &BlockContext,
         _account_tx_context: AccountTransactionContext,
     ) -> TransactionExecutionResult<(Fee, Option<CallInfo>, ResourcesMapping)> {
-        // The calldata includes the "from" field, which is not a part of the payload.
-        let l1_handler_payload_size = self.calldata().0.len() - 1;
+        let l1_handler_payload_size = self.calldata().0.len();
```

Comment:
is the `from` field part of the payload now ?

Reply:
mb